### PR TITLE
Download core from public location. Upgrade to latest sync beta.

### PR DESCRIPTION
**Pull request by cmelchior**
@realm/java

### CODE REVIEW

File: realm/realm-library/build.gradle [hunk 1]
```diff
@@ -413,12 +413,10 @@ task downloadCore() {
...
-            // CI artifacts are only available if on the internal network or VPN
-            def downloadUrl = "s3://realm-ci-artifacts/sync/${project.coreVersion}/android/realm-sync-android-${project.coreVersion}.tar.gz"
-
-            println "Downloading ${downloadUrl}"
-            exec {
-                commandLine 's3cmd', '-c', project.s3cfg, '-f', 'get', "${downloadUrl}", "${project.coreArchiveFile}"
+            download {
+                src "http://static.realm.io/downloads/sync/realm-sync-android-${project.coreVersion}.tar.gz"
+                dest project.coreArchiveFile
+                onlyIfNewer false
```
Comment (nhachicha) [inline realm/realm-library/build.gradle]: wouldn't this download the package each time? acting like a snapshots?

Comment (beeender) [inline realm/realm-library/build.gradle]: `shouldDownloadCore()` will check the SHA1 first. So this is correct.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 ## Internal
 
-* Upgraded to Realm Core 2.0.1 / Realm Sync 1.3-BETA
+* Upgraded to Realm Core 2.0.1 / Realm Sync 2.0-BETA
```
Comment (zaki50) [inline CHANGELOG.md]: `1.0.0-BETA-2.0`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,9 @@
 * Fixed a bug that `android.net.conn.CONNECTIVITY_CHANGE` broadcast caused `RuntimeException` if sync extension was disabled (#3505).
 * Fixed a bug that `android.net.conn.CONNECTIVITY_CHANGE` was not delivered on Android 7 devices.
 
+## Internal
+
+* Upgraded to Realm Core 2.0.1 / Realm Sync 2.0-BETA
```
Comment (zaki50) [inline CHANGELOG.md]: `1.0.0-BETA-2.0`?

Comment (cmelchior) [inline CHANGELOG.md]: Yup. Sync was upgraded yesterday

**Comment by zaki50** [comment]
We need to update `README.md` as well.

```
 * Download & install s3cmd (`brew install s3cmd` on Mac, `sudo apt-get install s3cmd` on Ubuntu).
 * Get `.s3cfg` file and put it in your home directory. If you'd like to put it other location, add `s3cfg=<path to .s3cfg>` in `~/.gradle/gradle.properties`.
```

**Comment by dalinaum** [comment]
@zaki50 You mean those two lines should be deleted? (It is just for checking, not an opinion.)

**Comment by zaki50** [comment]
@dalinaum exactly!

File: realm/realm-library/build.gradle (hunk 1 above)
Comment (nhachicha) [inline realm/realm-library/build.gradle]: but `onlyIfNewer` is set to `false` my understanding is that option will bypass the SHA1 check?

Comment (zaki50) [inline realm/realm-library/build.gradle]: No.  If `shouldDownloadCore ()` returns `false`, `download` itself is always skipped regardless of `onlyIfNewer`.

**Comment by zaki50** [comment]
retest this please

**Pull request merged**